### PR TITLE
Created the test case of adding the product to the cart.

### PR DESCRIPTION
**Pull request by Flower1111**
Created adding_to_cart.py module in oct/pages and adding_to_cart.py module in oct/tests/web.
1. adding_to_cart.py module in oct/pages was created for test case implementation of adding the product to the cart.
2. adding_to_cart.py module in oct/tests/web was created for implementation and running the test case. This test case checks the possibility of adding the product to the cart.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+version: '2'
```
Comment (extsoft) [inline docker-compose.yml]: @Flower1111 please remove this file from the commit

File: oct/pages/adding_to_cart.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+from selenium.webdriver import Remote
+from oct.pages.base import Page
+
+
+class AddingToCart(Page):
```
Comment (extsoft) [inline oct/pages/adding_to_cart.py]: @Flower1111 looks like this is a ProductPage, isn't it? Also, please do not hardcode a product. Instead, please pass it to the constructor.

File: oct/tests/web/adding_to_cart.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+from oct.pages.adding_to_cart import AddingToCart
+
+
+class AddingProductToCart(Testcase):
+    @test
+    def test(self, grid: str) -> None:
+        chrome: Remote = Chrome(grid)
+        adding_to_cart = AddingToCart(chrome)
+        adding_to_cart.open()
+        adding_to_cart.press_add_to_cart()
```
Comment (extsoft) [inline oct/tests/web/adding_to_cart.py]: @Flower1111 we must have an assert here. Please add.

**Comment by GoshkaEgoshka** [comment]
> > > I check it in oct/pages/adding_to_cart.py line 20.
> > 
> > 
> > but in line 20 you just click on element.
> 
> No)
> I click on the cart button with items to check that the product was added to the cart in oct/pages/addition_to_cart.py line 20.

You must use assert, which checks whether the product is added or not (true / false)

File: docker-compose.yml (hunk 1 above)
Comment (Flower1111) [reply docker-compose.yml]: > @Flower1111 please remove this file from the commit

Done.

File: oct/pages/adding_to_cart.py (hunk 2 above)
Comment (Flower1111) [reply oct/pages/adding_to_cart.py]: Done.

File: oct/tests/web/adding_to_cart.py (hunk 3 above)
Comment (Flower1111) [reply oct/tests/web/adding_to_cart.py]: Done.

File: oct/pages/adding_to_cart.py [hunk 4]
```diff
@@ -2,10 +2,11 @@
 from oct.pages.base import Page
 
 
-class AddingToCart(Page):
-    def __init__(self, browser: Remote):
+class ProductPage(Page):
```
Comment (GoshkaEgoshka) [inline oct/pages/adding_to_cart.py]: In master "Product page" was implemented. You need to add new func adding_to_cart

File: oct/tests/web/adding_to_cart.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+
+class AddingProductToCart(Testcase):
+    @test
+    def test(self, grid: str) -> None:
+        chrome: Remote = Chrome(grid)
+        adding_to_cart = ProductPage(chrome, "iMac")
+        adding_to_cart.open()
+        adding_to_cart.press_add_to_cart()
+        assert ProductPage(chrome, "iMac").loaded()
```
Comment (extsoft) [inline oct/tests/web/adding_to_cart.py]: @Flower1111 Why do we check that a product page is loaded? I think we need to check a cart.

Comment (Flower1111) [reply oct/tests/web/adding_to_cart.py]: Fixed it.

File: code-assessment.sh [hunk 6]
```diff
@@ -13,7 +13,7 @@ mypy oct suite.py || add_fail mypy
 
 if [[ ${#FAILURES[@]} -ne 0 ]]; then
     cat <<RESULT
-
+
```
Comment (extsoft) [inline code-assessment.sh]: @Flower1111 please reset this change.

File: oct/pages/product_page.py [hunk 7]
```diff
@@ -23,3 +23,9 @@ def open_brand_page(self) -> None:
             '//*[@id="content"]/div[1]/div[2]/ul[1]/li[1]/a'
         )
         brand_name.click()
+
+    def press_add_to_cart(self) -> None:
+        self._browser.find_element_by_id("button-cart").click()
+
+    def displaying_message_about_adding_product(self) -> bool:
+        return self._browser.find_element_by_xpath('//*[@id="product-product"]/div[1]')
```
Comment (extsoft) [inline oct/pages/product_page.py]: @Flower1111 An exception will be thrown if the element is not found. So, this method can return True or thrown an error. But I think if you can't find an element, you have to return False.

File: oct/pages/product_page.py [hunk 8]
```diff
@@ -23,3 +23,9 @@ def open_brand_page(self) -> None:
             '//*[@id="content"]/div[1]/div[2]/ul[1]/li[1]/a'
         )
         brand_name.click()
+
+    def press_add_to_cart(self) -> None:
+        self._browser.find_element_by_id("button-cart").click()
+
+    def displaying_message_about_adding_product(self) -> bool:
```
Comment (extsoft) [inline oct/pages/product_page.py]: @Flower1111 bad method name.

**Comment by mistergreen22** [comment]
Why you change 26 files?? I think you did something wrong, probably you should decline this request and understand what went wrong.

**Pull request closed without merging**